src/components/index.js

```diff
@@ -51,6 +51,7 @@ export { Modal } from './Modal'
 ////////////////////////////
 // ---- NORA EXPORTS ---- //
 ////////////////////////////
-export { IconLink } from '../nora/components/index'
 export { Header } from '../nora/components/index'
+export { IconLink } from '../nora/components/index'
+export { NoraButton } from '../nora/components/index'
```

Comment:
We have things named specifically for Nora in the EDS?   That seems like an odd direction.

Reply:
We have a whole sub-section for these Nora based components. The idea is that Design wants the main product sort of "locked up", and these Nora component are likely not appropriate for say the main funnel app. It's sort of best we have now sort of thing.

Reply:
So why even put them in the EDS?